src/main/java/org/aksw/simba/squirrel/analyzer/impl/GuessMimeType.java

```diff
@@ -0,0 +1,51 @@
+package org.aksw.simba.squirrel.analyzer.impl;
+
+import java.io.File;
+import java.io.FileInputStream;
+import org.aksw.simba.squirrel.analyzer.Guess;
+import org.apache.tika.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GuessMimeType implements Guess {

```

Comment:
`GuessMimeType` is not a "classic" class name because it does not describe an object but a command. Please give a better name (after enhancing the naming of the `Guess` interface).